cli/mender-artifact/partition.go

```diff
@@ -341,8 +374,8 @@ func (f *fatFile) Write(b []byte) (n int, err error) {
 	if err = f.tmpf.Sync(); err != nil {
 		return 0, errors.Wrap(err, "fatFile: Write: Failed to sync tmpfile")
 	}
-	// Use MTools to write to the fat-partition
-	cmd := exec.Command("mcopy", "-i", f.path, f.tmpf.Name(), "::/"+f.imageFilePath)
+	f.tmpf.Close()
```

Comment:
Why do we `Close`, here? We do have the close already in `func (f *fatFile) Close() (err error)` method